.github/workflows/artifact-engines.yml

```
@@ -52,3 +52,185 @@ jobs:
+          p50['name'] = 'Canonical Local Repository Path & Freshness Guard'
+          p50['type'] = 'SETUP + ANALYZE + DIRECTORY'
+          p50['class'] = 'STANDARD AI / CANONICAL LOCAL REPOSITORY INTAKE'
+          p50['sprintRole'] = 'Establish and reuse one canonical local checkout mapping per machine/execution profile, verify OS/shell/root/remote identity, and refresh remote truth before repository work'
+          p50['useWhen'] = 'The repository is known but agents may not know which box, operating system, shell, checkout, or worktree is canonical; a path is being guessed; multiple clones/worktrees exist; or local branch freshness is uncertain.'
+          p50['inspectFirst'] = 'Existing machine/profile path owners and local config; Hooks/Glossary/CODEBASE_MAP/AGENTS/trigger surfaces; actual OS/platform and shell; current directory; Git worktrees; candidate remotes; branch/tracking/default-branch state; remote freshness; and any operator-supplied path.'
+          p50['expectedOutput'] = 'One verified machine/execution-profile -> canonical repository-root mapping, one reused path authority, a lightweight discoverability pointer through an existing agent-facing route, root/remote/default-branch evidence, shell-correct entry commands, refreshed remote truth, and no duplicate checkout invented merely to continue work.'
+          p50['nextStep'] = 'If no matching checkout exists, run P61 and return here to register/verify the new checkout. If path discoverability infrastructure is absent and repository setup mutation is authorized, use P87 or the existing harness owner to install the smallest pointer to this canonical contract. After root and freshness gates pass, continue with the task-specific prompt without creating another clone or path registry.'
+          p50['proofGate'] = 'Exactly one canonical path authority is selected per machine/execution profile; the verified root matches the requested remote; OS/shell and worktree context are explicit; at least one existing Hooks/Glossary/CODEBASE_MAP/AGENTS/trigger front door points agents to the canonical path owner without duplicating absolute-path truth; git fetch --all --prune --tags refreshes reachable remote truth before branch-sensitive iteration; the actual remote default branch is resolved; clean behind-only state is fast-forwarded while dirty/diverged work is preserved; and no foreign-machine absolute path or unnecessary duplicate clone is treated as canonical.'
+          block = r'''
+
+CANONICAL LOCAL PATH CONTRACT
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)